payments/invoice/seamless-view.md

```diff
@@ -32,114 +34,9 @@ does not need to leave your webpage, since we are handling the payment in the
 
 ![screenshot of the invoice payment window][invoice-payment]{:height="425px" width="700px"}
 
-## Purchase Flow
-
-The sequence diagram below shows a high level description of the
-invoice process.
-
-```mermaid
-sequenceDiagram
-    Consumer->>Merchant: Start purchase
-    activate Merchant
-    note left of Merchant: First API request
-    Merchant->>-SwedbankPay: POST /psp/invoice/payments ①
-    activate SwedbankPay
-    SwedbankPay-->>-Merchant: rel: view-authorization ②
-    activate Merchant
-    Merchant-->>-Consumer: Display all details and final price
-    activate Consumer
-    note left of Consumer: Open iframe ③
-    Consumer->>Consumer: Input email and mobile number
-    Consumer->>-Merchant: Confirm purchase
-    activate Merchant
-    note left of Merchant: Second API request
-    Merchant->>-SwedbankPay: Post psp/invoice/authorization ④
-    activate SwedbankPay
-    SwedbankPay-->>-Merchant: Transaction result
-    activate Merchant
-    note left of Merchant: Third API request
-    Merchant->>-SwedbankPay: GET <payment.id> ⑤
-    activate SwedbankPay
-    SwedbankPay-->>-Merchant: payment resource
-    activate Merchant
-    Merchant-->>-Consumer: Display result
-```
-
-{% include alert.html type="neutral" icon="info" body="
-Note that the invoice will not be created/distributed before you have
-made a `capture` request." %}
-
-The Capture/Cancel/Reversal options are described in the [after payment
-section][after-payment]. The link will take you directly to the API descriptions
-for the requests.
-
-### Explainations
-
-* ① Start with collecting all purchase information and make a `POST` request
```

Comment:
Shouldn't we have this list below the diagram?

Reply:
Sure, I can put it back in :) Only the explanations or the API requests as well?

Reply:
Only the list